src/election.ts

```diff
@@ -88,13 +88,23 @@ export type Vote = CandidateVote | YesNoVote
 export type OptionalVote = Optional<Vote>
 export type VotesDict = Dictionary<Vote>
 
+export enum BallotType {
+  Standard = 0,
+  Absentee = 1,
+  Provisional = 2,
+}
+
+// Update this whenever a new BallotType case is added.
+export const BallotTypeMaximumValue = BallotType.Provisional
```

Comment:
One thing to note here: this will currently use two bits to encode a `BallotType` because the maximum is less than `2 ** 2`. However, it would be a backward-incompatible change if we were to add two more ballot type cases. Instead of doing that, I think we should pick a number of bits we're comfortable with and just encode that here. Four bits would allow up to 16 types, and I'm guessing that'd be sufficient. @benadida?

```suggestion
export const BallotTypeMaximumValue = (1 << 4) - 1
```